start_integration.py

```diff
@@ -7,28 +7,14 @@
 from sms_notifservice import SmsNotifService
 
 with open('settings.json', "rb") as SFile:
-    pwd_data = json.loads(SFile.read().decode('utf-8'))
-
-ov_url = pwd_data["oneVizionUrl"]
-ov_login = pwd_data["oneVizionLogin"]
-ov_pwd = pwd_data["oneVizionPwd"]
-service_id = pwd_data["serviceId"]
-phone_number_field = pwd_data["phoneNumberField"]
-
-awsAccessKeyId = pwd_data["awsAccessKeyId"]
-awsSecretAccessKey = pwd_data["awsSecretAccessKey"]
-awsRegion = pwd_data["awsRegion"]
-
-max_attempts = pwd_data["maxAttempts"]
-next_attempt_delay = pwd_data["nextAttemptDelay"]
+    settings_data = json.loads(SFile.read().decode('utf-8'))
 
 with open('ihub_parameters.json', "rb") as PFile:
     ihub_data = json.loads(PFile.read().decode('utf-8'))
 
-process_id = ihub_data['processId']
-log_level = ihub_data['logLevel']
-
-notification_service = SmsNotifService(service_id, ov_url, ov_login, ov_pwd, phone_number_field,
-                                       awsAccessKeyId, awsSecretAccessKey, awsRegion, max_attempts,
-                                       next_attempt_delay, process_id, log_level)
+notification_service = SmsNotifService(settings_data["serviceId"], settings_data["oneVizionUrl"], 
```

Comment:
I propose to place each parameter in the single line

Reply:
Done